src/azure-cli/azure/cli/command_modules/sqlvm/custom.py

```diff
@@ -171,14 +171,14 @@ def aglistener_update(instance, sql_virtual_machine_instances=None):
 
 # pylint: disable=too-many-arguments, too-many-locals, line-too-long, too-many-boolean-expressions
 def sqlvm_create(client, cmd, sql_virtual_machine_name, resource_group_name, sql_server_license_type,
-                 location=None, sql_image_sku=None, enable_auto_patching=None,
+                 location=None, sql_image_sku=None, enable_auto_patching=None, sql_management_mode="LightWeight",
```

Comment:
> sql_management_mode="LightWeight" [](start = 78, length = 34)

this is the by default value for create and update?

Reply:
Yes, we're setting LightWeight as default value for create. Are we changing that?